ob-splunk.el

```diff
@@ -28,14 +28,19 @@
 
 ;;; Code:
 (require 'ob)
+(require 'url-util)
+
+;;; editable splunk src code blocks
+(unless (assoc "splunk" org-src-lang-modes)
```

Comment:
If user is using another package like `pepita-mode` we probably don't want to override that association